src/compiler/filesystem_local.cc

```diff
@@ -80,7 +84,7 @@ bool FilesystemLocal::do_is_regular_file(const char* path) {
 }
 
 bool FilesystemLocal::do_is_directory(const char* path) {
-#ifndef TOIT_LINUX
+#ifndef USE_XSTAT64
```

Comment:
I would probably flip this around and use the positive (here and above):

```
#ifdef USE_XSTAT64
   xstat64
#else 
   ... stat
```

Reply:
Sure, I can update that quick.